packages/examples/src/examples/ruleInheritance.ts

```diff
@@ -93,12 +94,36 @@ export const data = {
   toggleControl: true
 };
 
+const actions = [
+  [
```

Comment:
Currently the actions of ruleInheritance aren’t working. I think you added one pair of braces to many.
```suggestion
```